.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,3 +38,4 @@ implementation proposal in the referenced issues.
 - [ ] Documentation: Relevant documentation is added or updated.
 - [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
 - [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.
+- [ ] Target: The changes do not contain API updates or require a DB Migration and is hence targeting `master`.
```

Comment:
I would reword this to:

Target: If the changes are not changing any API, configuration and storage, the target branch is set to `master`, otherwise the target branch is set to `develop`.

Reply:
done